src/api/types.ts

```diff
@@ -42,3 +43,21 @@ export interface AnnotationsAPI {
 
     destroy(): void;
 }
+
+export type CollaboratorsRequestData = {
```

Comment:
It looks like this can be declared inline in the `CollaboratorsAPI` type, since it's not reused.